.github/containers/ruby/repo/packages/flux-sched/package.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
I am not sure we can have the spack license in our repo. This applies to all packages you are including.

Reply:
changed to AMS